OpenRA.Game/Network/OrderManager.cs

```diff
@@ -55,6 +55,8 @@ public sealed class OrderManager : IDisposable
 
 		public IReadOnlyList<TextNotification> NotificationsCache => notificationsCache;
 
+		public readonly Dictionary<int, bool> MutedPlayerLib = new Dictionary<int, bool>();
```

Comment:
Two things, the naming, would MutedPlayers be a better name? Also this doesn't belong on the ordermanager.

Reply:
I agree, but I'm not sure where it should belong. Order manager just fits with the usecase. I also wanted to tackle the dependancies first before going into this pr

Reply:
Can this live in `TextNotificationManager`?

Reply:
> Can this live in `TextNotificationManager`?

No, it is static and I don't want this value to persist between games

Reply:
This is a bigger problem in the engine structure, that i have mentioned in the past, and causes us to add logic to classes by the life span of them and not where it really belongs, But it would be nice if we dont add more stuff here when i already have spent some time making this class much more cleaner.

Reply:
I understand that and fully agree. This was just a temporary way to get mutes to work and I will find a better place for it. Having said that it would be best to focus on merging dependancies before moving onto this pr

Reply:
> No, it is static and I don't want this value to persist between games

`IngameChatLogic` would need to clear it in `Disposed` which I assume runs when the game ends?

Reply:
> > No, it is static and I don't want this value to persist between games
> 
> `IngameChatLogic` would need to clear it in `Disposed` which I assume runs when the game ends?

It runs when the game ends, and when player exist the game. So basically mutes would disappear when you enter after-game chat, what is definitely not desired